pkg/testutil/testutil.go

```diff
@@ -21,20 +20,12 @@ func CaptureStdout(f func()) string {
 	}()
 	os.Stdout = writer
 	log.SetOutput(writer)
-	out := make(chan string)
-	wg := new(sync.WaitGroup)
-	wg.Add(1)
-	go func() {
-		var buf bytes.Buffer
-		wg.Done()
-		_, err = io.Copy(&buf, reader)
-		if err != nil {
-			panic(err)
-		}
-		out <- buf.String()
-	}()
-	wg.Wait()
 	f()
 	_ = writer.Close()
```

Comment:
Put it in defer, it looks more reasonable

Reply:
@mumoshu it will cause the code blocked forever.

Reply:
@yxxhero Which part of code hangs, you mean? If you write without reading, writes end up hanging. Similarly, if you keep reading without closing the Writer, the reader ends up hanging forever(due to it waits until the Writer closes).